### ISSUE TITLE
UX/UI - Event Listings - sold out events

### ISSUE THREAD

**Issue by CheetoMao** [revelrylabs/bn-mobile-react#782]
## Background
Currently, users can view events, tap to navigate to a specific event details page, tap to purchase tickets for that event AND THEN can see that the event is completely sold out.

We should surface this information sooner in the workflow to avoid user frustration.

**Proposed enhancements:**
- add a 'sold out' badge to sold out events on the listing view and the event details view
- allow users to toggle on/off viewing sold out events

**Comment by iamjoshfrank** [comment]
I agree - to start, "Sold Out" should appear in place of the ticket price(s) listed on the "Explore" screen and the "Event Details" screen.

I see 2 separate Issues here:
1. Displaying "sold out"
2. Toggle or other mechanism to hide "sold out" events

I'll loop back around to split these out, but anyone on the team can beat me to it if they're looking for something new.
